test/models/chan.js

```diff
@@ -32,9 +32,7 @@ describe("Chan", function() {
 		};
 
 		var getUserNames = function(chan) {
-			return chan.users.map(function(u) {
-				return u.name;
-			});
+			return chan.users.map(u => u.nick);
```

Comment:
Not true: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/map?v=control

Also, even if it can physically only happen with 1, there's no reason not to have them, as it just makes things safer. Just like why we do ';' and {} around things that don't technically need them. It's to help us, because we make mistakes.